server.js

```diff
@@ -146,7 +146,7 @@ app.get('/api/info', function(req, res) {
 app.get('/api/auth/:path', function(req, res) {
   auth.checkUrl(req, function(err, output) {
     if (err) {
-      return serverUtils.serverError(err, req, res);
+      return serverUtils.error(err, req, res);
```

Comment:
I think it makes sense, one function reports errors details/messages and the other doesn't (I assume to avoid leaking sensitive data) but maybe the naming is confusing... I will have a look.